tools/lsp/preview.rs

```diff
@@ -985,6 +985,11 @@ async fn parse_source(
         cc.resource_url_mapper = resource_url_mapper();
     }
 
+    if std::env::var_os("SLINT_ENABLE_EXPERIMENTAL_FEATURES_INSIDE_PREVIEW").is_some() {
```

Comment:
Why do we need an extra env variable?
I think it's fine to re-use `SLINT_ENABLE_EXPERIMENTAL_FEATURES` isn't it?

Reply:
Why is that?

> It's not as if we paid extra for the env vars :-)
 
Yes we do. We pay in complexity.
Our user wanting to try out experimental features will have to set two environment variables. and so we'll have to explain with more details.